src/_common/video/player/player.ts

```diff
@@ -318,7 +318,15 @@ export default class AppVideoPlayer extends Vue {
 	syncWithState() {
 		if (this.player.queuedFullScreenChange !== null) {
 			if (this.player.queuedFullScreenChange) {
-				this.$el.requestFullscreen();
+				if (this.player.altControlsBehavior) {
+					const video: any = this.$el.getElementsByTagName('video')[0];
```

Comment:
I would try to make this safer just in case there is no video element for whatever reason. Maybe just use `querySelector` and check for falsey.

Reply:
There's no way to do this through a vue ref, is there?

Reply:
We don't have access to the direct video tag here. Changed from `getElementsByTagName` to `querySelector`.